samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -57,9 +57,7 @@ static ALWAYS_INLINE void rwup_if_suspended(void)
 
 static void input_cb(struct input_event *evt)
 {
-	uint8_t tmp[MOUSE_REPORT_COUNT];
-
-	(void)memcpy(tmp, report, sizeof(tmp));
+	static uint8_t tmp[MOUSE_REPORT_COUNT];
```

Comment:
Why `static`? `k_msqg_put` is copying the data.

Reply:
To keep the state of the buttons, otherwise there would be a button state change on x/y event.